.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+## What type of PR is this? (check all applicable)
+
+- [ ] Refactoring existing code
+- [ ] Adding tests to existing code
+- [ ] New Feature
+- [ ] Bug Fix
+- [ ] Optimization
+- [ ] Documentation Update
+
+## Description of the change
+
+## Related Issues & Documents (if needed)
+
+## QA Instructions(if necessary), Screenshots of Output or Recordings
+
+_Please replace this line with instructions on how to test your changes
+
+## Added tests?
+
+- [ ] Yes
+- [ ] No, because they aren't needed
+- [ ] No, because I need help
+
+## Added to documentation / readme?
```

Comment:
Added to documentation / readme / updated ```requirements.txt```(if needed) ?

Reply:
Made the changes, please check !